### ISSUE TITLE
Show unimplemented cards as playable?

### ISSUE THREAD

**Issue by JayDi85** [magefree/mage#9121]
From #9103 discussion.

There are possible to add unimplemented cards as simple/bulk card, so users can add it to deck, play and do all other things. It can be simple colored creature or land card with additional rules about unimplemented feature.

One of the annoying thing in current version -- if you download new deck with unimplemented card then you must edit deck and replace it with another card.

So is a playable unimplemented cards can be useful?

**Comment by jeffwadsworth** [comment]
Looks like a reasonable idea to me.

**Comment by the-red-lily** [comment]
Are you suggesting that the card's abilities be completely ignored? Or are we going to somehow allow players to manually bypass the game engine to follow a card's rules text?

Either way it seems useful. The first option is easier to implement, of course :)

**Comment by JayDi85** [comment]
I'm talking about a minimum impact to the game. Just empty card. XMage
doesn't support manual/user defined rules by design.

**Comment by Alex-Vasile** [comment]
Yes, I like this. Though the user should still be given a warning about having unimplemented cards in their deck.

A basic land card with extra text I think would be best. This way the checks for singleton formats won't need a special check about duplicate cards.
